physical_infra_provider_configuration/configuration/topics/setting_up_lxca.adoc

```diff
@@ -0,0 +1,9 @@
+=== Setting up Lenovo XClarity Administrator
+
+Physical-infrastructure providers are used to connect CloudForms to one or more Lenovo XClarity Administrator instances. Lenovo XClarity Administrator is a centralized, resource-management solution that simplifies infrastructure management, speeds responses, and enhances the availability of Lenovo® server systems and solutions. It runs as a virtual appliance that automates discovery, inventory, tracking, monitoring, and provisioning for server, network, and storage hardware in a secure environment.
```

Comment:
These two paragraphs do not seem to fit in this section. I think we should remove them altogether and just leave the last couple of sections that reference downloading LXCA and the Pro- license Note.